src/routes.js

```diff
@@ -68,14 +73,21 @@ const utxoForAddresses = (dbApi: DbApi, { logger, apiConfig }: ServerConfig) =>
 ) => {
   validateAddressesReq(apiConfig.addressesRequestLimit, req.body)
   logger.debug('[utxoForAddresses] request is valid')
-  const result = await dbApi.utxoForAddresses(req.body.addresses)
+  const utxos: Array<UtxoForAddressesDbResult> = await dbApi.utxoForAddresses(req.body.addresses)
```

Comment:
I would prefer to append the tokens to the response at dbApi level - it may not be a pure SQL query but on the high level the concern here is returning data from the database and utxos without tokens don't make that much sense as they are unspendable without that information